### PR TITLE
Update `arkworks` dependencies

### PR DESCRIPTION
**Pull request by tessico**
## Description

Pay special attention to:
- the few `TODO`s I left
- usage of Projective/Affine representation
- (de)serialization: compressed/uncompressed (flags are sometimes ignored) & checked/unchecked
- creation of cosets for (i)FFTs

closes: https://github.com/EspressoSystems/jellyfish/issues/45

---

Before we can merge this PR, please make sure that all the following items have been
checked off. If any of the checklist items are not applicable, please leave them but
write a little note why.

- [x] Targeted PR against correct branch (main)
- [x] Linked to GitHub issue with discussion and accepted design OR have an explanation in the PR that describes this work.
- [ ] Wrote unit tests
- [ ] Updated relevant documentation in the code
- [ ] Added a relevant changelog entry to the `Pending` section in `CHANGELOG.md`
- [ ] Re-reviewed `Files changed` in the GitHub PR explorer

### CODE REVIEW

**Review by mrain — COMMENTED** [review]
Interestingly, for serialization, arkworks currently does the same whether it's compressed/uncompreseed or checked/unchecked.

**Review by mrain — DISMISSED** [review]

File: plonk/src/proof_system/structs.rs [hunk 1]
```diff
@@ -100,7 +100,7 @@ where
                 .chunks_exact(2)
                 .map(|chunk| {
                     if chunk.len() == 2 {
-                        Commitment(Affine::new(chunk[0], chunk[1], false))
```
Comment (alxiong) [inline plonk/src/proof_system/structs.rs]: I wonder if we should use this constructor or `Affine::new_unchecked` ( cuz the former include `is_on_curve` and `is_in_correct_subgroup` by default)

**Review by alxiong — DISMISSED** [review]
Also gone through it. LGTM! 🚀 

Again, really appreciate the work Marti @tessico

**Review by alxiong — APPROVED** [review]

**Comment by tessico** [comment]
I'm glad to see this merged! Just saw that since https://github.com/EspressoSystems/tagged-base64/pull/47 hasn't been merged and tagged yet, the master here now depends on a branch of tagged-base64, which probably isn't ideal for long term, sth to keep in mind :)

**Pull request merged**